Directory.Build.props

```diff
@@ -0,0 +1,29 @@
+﻿<Project>
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+    <!-- warning NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead. -->
+    <NoWarn>NU5125;$(NoWarn)</NoWarn>
```

Comment:
Suppressing this warning should no longer be needed since I switched to using PackageProjectUrl.